exonum_launcher/launcher.py

```diff
@@ -143,10 +147,15 @@ def start_all(self) -> None:
                 self.launch_state.complete_initialization(instance, ActionResult.Fail)
                 continue
 
-            config_loader = self._artifact_plugins.get(instance.artifact, DefaultInstanceSpecLoader())
-            start_request = self._supervisor.create_start_instance_request(instance, config_loader)
+        config_loaders = [
+            self._artifact_plugins.get(instance.artifact, DefaultInstanceSpecLoader())
+            for instance in self.config.instances
+        ]
+        start_request = self._supervisor.create_start_instance_request(self.config.instances, config_loaders,
+                                                                       self.config.actual_from)
 
-            txs = self._supervisor.send_start_instance_request(start_request)
+        txs = self._supervisor.send_start_instance_request(start_request)
+        for instance in self.config.instances:
```

Comment:
I think it could be one initialization, since there is only one request. However, it could be fixed in other PR, so please just add a TODO for it.

Reply:
I thought of it and decided to leave it like this for now, since there is still a check for instance IDs that were started.

Reply:
That's why I suggest to add a TODO :) Later we'll be able to find the better soluiton, for now it's OK.